deploy/basic/terraform/atp.tf

```diff
@@ -9,7 +9,7 @@ resource "oci_database_autonomous_database" "mushop_autonomous_database" {
   compartment_id           = var.compartment_ocid
   cpu_core_count           = 1
   data_storage_size_in_tbs = 1
-  db_name                  = "${var.database_name}${random_id.mushop_id.dec}"
+  db_name                  = var.database_name
```

Comment:
We included originally because we where testing and had greater limits. Always Free resource is actually limited to two, but there's no way to install two mushop-basic on the same tenancy today.